src/components/ArticleById.jsx

```diff
@@ -30,6 +36,18 @@ export default function ArticleById() {
           <br></br>
           {article.created_at}
         </h4>
```

Comment:
I'd consider formatting this to remove the T, Z and seconds from the timestamp using JS.

Reply:
Thank you, Eli. I changed it just now!